### DISCUSSION TITLE
Why file size increased after compressing textures?

### DISCUSSION THREAD

**Discussion by dasmehdix** [donmccurdy/glTF-Transform#871] [General]
Hi all. I have tried to compress textures with KTX support. Some models can be compressed perfectly, file size and memory usage decreases sharply. But some models file size is increased after KTX compression. I was using `etc1s` option. 
Example convert command:
`gltf-transform etc1s example.glb example_ktx.glb --quality 255`

Why this happens? How can I overcome this? As far as I know, `etc1s` option with KTX support should decrease file size and ram usage.

**Comment by donmccurdy** [comment]
Hi @dasmehdix! KTX2 is always smaller in VRAM than JPEG or PNG. KTX2 is often smaller on disk than PNG or JPEG, but there's no guarantee here, and different images may compress better with one method than another. If you care more about disk size than VRAM, then WebP or AVIF would be the best choices for file size on disk.
